### ISSUE TITLE
a question about evaluate_FlowFormer.py

### ISSUE THREAD

**Issue by deqings** [drinkingcoder/FlowFormer-Official#20]
Very interesting work!

I have a question about evaluate_FlowFormer.py, which reports AEPE 0.63/1.51 on Sintel clean/final for the provided things.pth checkpoint. 
            flow_pre = model(image1, image2)
            flow_pre = padder.unpad(flow_pre[0]).cpu()
            epe = torch.sum((flow_pre - flow_gt)**2, dim=0).sqrt()
For my run on Sintel images, flow_pre is of size [1, 2, 436, 1024] and flow_gt [2, 436, 1024]. (flow_pre - flow_gt)**2 is of size [1, 2, 436, 1024] and sum over dim 0 results in size [2, 436, 1024] for epe. Thus, "epe" doesn't sum over the horizontal and vertical flow errors. 
            flow_pre = torch.squeeze(padder.unpad(flow_pre[0]).cpu())
would results in size [436,1024] for epe and reports AEPE 1.01/2.40. 

Could you please help check it?

Thanks.

**Comment by btwbtm** [comment]
Hi, thanks for your interest in our work. 
https://github.com/drinkingcoder/FlowFormer-Official/blob/main/evaluate_FlowFormer.py#L55-L65
In Line56-57, image1 and image2 are unsqueezed to have shape [1,2,436,1024], while flow_gt is of [2, 436,1024]. In Line 63, we use flow_pre[0] to squeeze flow_pre so that it has shape [2, 436, 1024], which should be the same as flow_gt. Could you double check and let me know if I am wrong.

**Comment by drinkingcoder** [comment]
Thanks for pointing that out. Sorry for the mistake :dizzy_face:. We will roll back our Arxiv paper to the ECCV submission version, where the reported AEPE is 0.95/2.35, calculated through the tile technique.

**Comment by deqings** [comment]
Thanks for checking it. These are still impressive numbers.

BTW: if you haven't submit your new ArXiv paper, could you please also correct the three A+S+K+H entries in Table 1? The numbers there are for the training set of Sintel and KITTI after the models have been trained on AutoFlow. Thanks.

**Comment by drinkingcoder** [comment]
Many many thanks for your reminding!  We will update that!
